app/helpers/application_helper.rb

```diff
@@ -66,4 +66,9 @@ def genre_links(options = {})
   def display_duration(options = {})
     options[:value]&.map { |v| v.scan(/([0-9]{2})/).join(':') }
   end
+
+  # To render format icon on search results as the default thumbnail for now
+  def render_thumbnail(document, _options = {})
```

Comment:
Curious about why you're using document as opposed to options (i.e., `options[:value]`)

Reply:
I could not get it working with options[:value] :(

Reply:
I guess because of the scope at which this is called. Okay, learned something new! :)